pages/patterns/create-a-profile/address.md

```diff
@@ -74,13 +74,21 @@ Provide all the fields needed for a user to provide a physical address, even les
 </div>
 
 <div class="usa-accordion usa-accordion--bordered site-accordion-code site-component-preview margin-top-2">
-  <button class="usa-accordion__button" aria-controls="accordion-preview-01" aria-expanded="true"><h4 id="pattern-preview-physical-address">Pattern preview</h4></button>
+  <h4 id="pattern-preview-physical-address">
```

Comment:
### Missing accordion classes

Heading is missing the accordion classes, causing a gray accordion button instead of blue

```suggestion
  <h4 id="pattern-preview-physical-address" class="usa-accordion__heading site-accordion-heading">
```